integration_test/cases/type.exs

```diff
@@ -203,6 +203,14 @@ defmodule Ecto.Integration.TypeTest do
            [%{"foo" => "http://foo.com", "bar" => "http://bar.com"}]
   end
 
+  @tag :map_type
+  test "typed float map" do
+    post = TestRepo.insert!(%Post{intensities: %{"foo" => 1.0, "bar" => 416500.0}})
+
+    assert TestRepo.all(from p in Post, where: p.id == ^post.id, select: p.intensities) ==
+           [%{"foo" => 1.0, "bar" => 416500}]
+  end
```

Comment:
Can you add a test that loads a value that was persisted as a string? This test relies on the implementation detail of postgres json.

Reply:
@ericmj This test is wrong, it is meant to return float. :(